src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const orderedStyles = sourceString.split(';');
+
+  const cssObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `cssObject`. It's better to use more descriptive names that reflect the content of the variable and its purpose within the function.